### PR TITLE
First Checkin

### PR DESCRIPTION
**Pull request by BrittanyWalters**

### CODE REVIEW

**Review by idiglove — COMMENTED** [review]
im taking a look :)

File: src/components/DisneySearch/components/Body/components/CharacterCard/CharacterCard.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import { Box } from "@mui/material";
+import { Character } from "../../../../../../hooks/useCharacters";
```
Comment (idiglove) [inline src/components/DisneySearch/components/Body/components/CharacterCard/CharacterCard.tsx:2]: use import alias here

File: src/components/DisneySearch/components/Body/components/CharacterCard/CharacterCard.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                            <div className="font-semibold">
+                                Featured Short Films
+                            </div>
+                            <div className="featured">
+                                {character.shortFilms.join(", ")}
+                            </div>
+                        </>
+                    ) : (
+                        <></>
+                    )}
```
Comment (idiglove) [inline src/components/DisneySearch/components/Body/components/CharacterCard/CharacterCard.tsx:70]: having too many conditions in render function is bad smell. Try making small components and have early returns

if (condition1) {
return <Component1>
}

if (condition2){
return <Component2>
}

File: src/components/DisneySearch/components/Body/components/CharacterDetails/CharacterDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                            title="Short Films"
+                            media={characterDetail.shortFilms}
+                        ></ShowMediaList>
+                    )}
+                    {!!characterDetail.tvShows.length && (
+                        <ShowMediaList
+                            title="TV Shows"
+                            media={characterDetail.tvShows}
+                        ></ShowMediaList>
+                    )}
```
Comment (idiglove) [inline src/components/DisneySearch/components/Body/components/CharacterDetails/CharacterDetails.tsx:55]: this file is doing too many things and has too many concepts. Apply SRP - single responsibility principle

File: src/components/DisneySearch/components/Header/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                    <img src={logo} className="logo" />
+                </a>
+            </div>
+            <div className="flex-1 mx-8">
+                <TextField
+                    sx={{
+                        "& .MuiOutlinedInput-root": {
+                            borderRadius: "20px",
+                        },
+                    }}
```
Comment (idiglove) [inline src/components/DisneySearch/components/Header/Header.tsx:35]: try to use classes prop instead

**Review by idiglove — CHANGES_REQUESTED** [review]
Here's my review, Im still not finished, but you can go with this for now. Bit busy now :D

**Comment by BrittanyWalters** [comment]
> Here's my review, Im still not finished, but you can go with this for now. Bit busy now :D

Great comments. Thank you so much!! They all make sense. When you do have a moment I'm really curious what your thoughts are how i setup the custom hook usecharacters. I'll start with your initial comments. You don't know how thankful I am.

**Comment by idiglove** [comment]
I'm happy that I made someone happy :D Chat me on discord lithia777

**Pull request merged**